main.py

```diff
@@ -42,6 +42,22 @@ def __init__(self, data, namespace, name=None):
         self.created_at = str(datetime.utcnow())
         self.owner = None
 
+    @property
+    def name(self):
+        log.warn(
```

Comment:
Please, change "warn" to "warning" and remove the PendingDeprecationWarning parameter.

Reply:
Done